frontend/src/Explorer/Util/String.js

```diff
@@ -9,3 +9,9 @@ exports.substituteImpl = function(str) {
     });
   };
 };
+
+exports.take = function (n) {
```

Comment:
Hm, that doesn't seem right. @sectore can you check that out when you get free time?

maybe we just had to use different `purescript-strings` library - like `npm i purescript-strings#v2.0.1 --save`